src/condor_ce_run

```diff
@@ -88,6 +88,19 @@ def parse_opts():
 
     return opts, args
 
+
+def run_command_debug(cmd, debug=False):
+    """Run cmd and print stdout/stderr if debug
+    """
+    rc, stdout, stderr = ce.run_command(cmd)
+
+    if debug:
+        print(stdout)
+        print(stderr, file=sys.stderr)
```

Comment:
Think about printing this to stdout instead of stderr? In some circumstances (ie. piped output, Condor jobs with no error file defined) the stderr output can get dropped.

Reply:
This is just taking the stderr from condor commands like `condor_submit` so it seems appropriate to just pass it through to stderr.